apps/src/templates/page.html.ejs

```diff
@@ -78,6 +78,7 @@
 </div>
 
 <div id="visualizationResizeBar">
+  <i id="visualizationResizeIcon" class="fa fa-ellipsis-v"></i>
```

Comment:
One more question: typically, for proper design encapsulation, we only reference classes in apps that are defined by CSS in the apps directory. I can't find `fa-ellipsis-v` - is that defined somewhere else?